flink-python/pyflink/table/tests/test_table_environment_api.py

```diff
@@ -275,16 +275,16 @@ def test_create_table_environment_with_blink_planner(self):
 
         self.assertEqual(
             planner.getClass().getName(),
-            "org.apache.flink.table.planner.StreamPlanner")
+            "org.apache.flink.table.planner.delegation.StreamPlanner")
 
         t_env = StreamTableEnvironment.create(
-            environment_settings=EnvironmentSettings.new_instance().use_blink_planner().build())
+            environment_settings=EnvironmentSettings.new_instance().use_old_planner().build())
```

Comment:
Why change to old? test name is `test_create_table_environment_with_blink_planner`? maybe should change method name?

Reply:
The test is trying to cover all 3 situations, 1) default behavior 2) use blink planner explicitly 3) use old planner explicitly.